.github/workflows/verify.yml

```
@@ -45,6 +45,27 @@ jobs:
     - name: Install dependencies (Nim)
       run: sudo apt install nim
 
+    - name: Set up Rust (1.42.0)
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: 1.42.0-x86_64-unknown-linux-gnu
+        default: true
+        profile: minimal
+
+    # required by cargo-udeps
+    - name: Set up Rust (nightly)
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: nightly-x86_64-unknown-linux-gnu
+        default: true
+        profile: minimal
+
+    # required only if you set `languages.rust.list_dependencies_backend.kind` to `"cargo-udeps"`
+    - name: Install cargo-udeps for Rust
+      uses: actions-rs/install@v0.1
+      with:
+        crate: cargo-udeps
+        use-tool-cache: true
```

@kmyk actions-rs/installはunmaintainedだし、結局他に良い方法が思い浮かばなかったのでGitHub Releasesからダウンロードする方法に戻していいでしょうか。

(`cargo install`すると4分、`--debug`付きでも90秒ほどかかります。)

```suggestion
      run: |
        version="$(curl https://crates.io/api/v1/crates/cargo-udeps | jq -r '.crate.max_version')"
        wget "https://github.com/est31/cargo-udeps/releases/download/v$version/cargo-udeps-v$version-x86_64-unknown-linux-gnu>
        tar xzvf "./cargo-udeps-v$version-x86_64-unknown-linux-gnu.tar.gz" -C /tmp/
        echo "/tmp/cargo-udeps-v$version-x86_64-unknown-linux-gnu" >> $GITHUB_PATH
```

---

@qryxip 戻した方がよさそうということならお願いします。このプルリクはもうマージしてしまったので、新しいプルリクを作ってください